src/main/java/car/Car.java

```diff
@@ -0,0 +1,28 @@
+package car;
+
+import utils.NumberGenerator;
+
+public class Car {
+    private final static int INITIAL_POSITION = 0;
+    private int position = INITIAL_POSITION;
+    private final static int BASE_NUMBER = 4;
```

Comment:
클래스내에 정의를 내리실때, 연관있는것들끼리 묶어두는게 좋습니다.

```suggestion
    private final static int INITIAL_POSITION = 0;
    private final static int BASE_NUMBER = 4;
        
    private int position = INITIAL_POSITION;
```

대부분 상수, 인스턴스 변수, 생성자, 메서드 이러한 순서로 두긴 하죠~